src/utils/MultiInviter.ts

```diff
@@ -216,62 +216,68 @@ export default class MultiInviter {
 
                 let errorText: string;
                 let fatal = false;
-                switch (err.errcode) {
-                    case "M_FORBIDDEN":
-                        if (isSpace) {
-                            errorText = _t('You do not have permission to invite people to this space.');
-                        } else {
-                            errorText = _t('You do not have permission to invite people to this room.');
-                        }
-                        fatal = true;
+                for (const errcode of [err.unstableErrcode, err.errcode]) {
```

Comment:
Any chance we can have some unit tests of this functionality? Or is it already covered (I couldn't find anything)

Reply:
It's not tested, but I'd be fine with doing so if I can figure it out :)

Reply:
Brilliant, thank you. #element-dev might have suggestions if it proves tricky.